File: PRAK305-2310817210026-MUHAMMADPUTRAAZKYALFANI.py
for i in range(0, 5):
    detik = int(input())
    jam = detik // 3600
    menit = (detik % 3600) // 60
    detik = detik % 60
    if jam >= 24:
        hari = jam // 24
        jam = jam % 24
        print(f'{hari} hari {jam:02}:{menit:02}:{detik:02}')
    else:
        print(f'{jam:02}:{menit:02}:{detik:02}')

File: PRAK305-2310817210026-MUHAMMADPUTRAAZKYALFANI.c
#include <stdio.h>
int main(){
    int d;
    for(int i=0;i<5;i++){
    scanf("%d", &d);
    int hari = d / 84600;
    int jam = d / 3600;
    int menit = (d - jam*3600) / 60;
    int detik = d - jam*3600 - menit*60;
    if(hari >= 1){printf("%d hari %2d:%2d:%2d\n", hari, jam - 24, menit, detik);}
    else{printf("%.2d:%.2d:%.2d\n", jam, menit, detik);}
    }
    return 0;
}